resources/router/security/localhost.pem

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
+MIIDhTCCAm2gAwIBAgIJAJrwNDwLlCFQMA0GCSqGSIb3DQEBCwUAMGQxCzAJBgNV
```

Comment:
change the file name to mg.key

Reply:
fixed via https://github.com/wso2/product-microgateway/pull/1525/commits/ea0fb028303ed596ba97d346872706aed1fb410e